pkg/ingestor/verifier/verifier_test.go

```diff
@@ -127,7 +129,10 @@ func (m *mockSigstoreVerifier) Type() VerifierType {
 
 func TestVerifyIdentity(t *testing.T) {
 	ctx := logging.WithLogger(context.Background())
-	err := RegisterVerifier(newMockSigstoreVerifier(), "sigstore")
+	var err error
+	once.Do(func() {
+		err = RegisterVerifier(newMockSigstoreVerifier(), "sigstore")
+	})
```

Comment:
synchronization is not the the issue. i don't think this is the right way to solve the issue here see my comment in https://github.com/guacsec/guac/issues/1336

Reply:
i see in this case its telling it to do it once, but I think it'll be nicer to have it be proper startup/teardown. Because it doesn't quite make sense in terms of the test itself

Reply:
@lumjjb How would we handle this? What would be the setup and teardown process for this? Please let me know.